about.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About - DBK Salon</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="about-style.css">
+</head>
+<body>
+  <div class="background-image" style="height: 25vh;">
```

Comment:
you have a height of 25vh inline and a height of 1100px in your css file. which one do you want?

Reply:
nevermind. i see what you're doing! this is for the about page.